src/openforms/contrib/objects_api/setup_configuration/tests/test_objects_api_config.py

```diff
@@ -0,0 +1,224 @@
+from pathlib import Path
+
+from django.test import TestCase
+
+from django_setup_configuration.test_utils import execute_single_step
+from zgw_consumers.constants import APITypes, AuthTypes
+from zgw_consumers.models import Service
+from zgw_consumers.test.factories import ServiceFactory
+
+from openforms.contrib.objects_api.models import ObjectsAPIGroupConfig
+from openforms.contrib.objects_api.tests.factories import ObjectsAPIGroupConfigFactory
+
+from ..steps import ObjectsAPIConfigurationStep
+
+TEST_FILES = (Path(__file__).parent / "files").resolve()
+CONFIG_FILE_PATH = str(TEST_FILES / "setup_config_objects_api.yaml")
+CONFIG_FILE_PATH_REQUIRED_FIELDS = str(
+    TEST_FILES / "setup_config_objects_api_required_fields.yaml"
+)
+CONFIG_FILE_PATH_ALL_FIELDS = str(
+    TEST_FILES / "setup_config_objects_api_all_fields.yaml"
+)
+
+
+class ObjectsAPIConfigurationStepTests(TestCase):
+    maxDiff = None
+
+    def setUp(self):
+        super().setUp()
+
+        self.objecttypes_service = ServiceFactory.create(
+            slug="objecttypen-test",
+            label="Objecttypen API test",
+            api_root="http://localhost:8001/api/v2/",
+            api_type=APITypes.orc,
+            auth_type=AuthTypes.api_key,
+            header_key="Authorization",
+            header_value="Token foo",
+        )
+        self.objects_service = ServiceFactory.create(
+            slug="objecten-test",
+            label="Objecten API test",
+            api_root="http://localhost:8002/api/v2/",
+            api_type=APITypes.orc,
+            auth_type=AuthTypes.api_key,
+            header_key="Authorization",
+            header_value="Token bar",
+        )
+        self.drc_service = ServiceFactory.create(
+            slug="documenten-test",
+            label="Documenten API test",
+            api_root="http://localhost:8003/documenten/api/v1/",
+            api_type=APITypes.drc,
+            auth_type=AuthTypes.zgw,
+            client_id="test_client_id",
+            secret="test_secret_key",
+        )
+        self.catalogi_service = ServiceFactory.create(
+            slug="catalogi-test",
+            label="Catalogi API test",
+            api_root="http://localhost:8003/catalogi/api/v1/",
+            api_type=APITypes.ztc,
+            auth_type=AuthTypes.zgw,
+            client_id="test_client_id",
+            secret="test_secret_key",
+        )
```

Comment:
why is this done in `setUp` rather than `setUpTestData`?

Reply:
that was an oversight, I'll move it to `setUpTestData` since these services are only referred to and not modified by any tests